code/Controllers/CMSMain.php

```diff
@@ -1038,56 +1038,93 @@ public function getBreadcrumbsBackLink()
             ->Link;
     }
 
+    /**
+     * Generate a breadcrumb for based on the page currently being viewed.
+     * @param bool $unlinked Wheter to include links
+     * @return ArrayList<ArrayData>
+     */
     public function Breadcrumbs($unlinked = false)
     {
         $items = new ArrayList();
 
-        if ($this->TreeIsFiltered()) {
-            $items->push(new ArrayData([
-                'Title' => CMSPagesController::menu_title(),
-                'Link' => ($unlinked) ? false : $this->LinkPages()
-            ]));
-            $items->push(new ArrayData([
-                'Title' => _t('SilverStripe\\CMS\\Controllers\\CMSMain.SEARCHRESULTS', 'Search results'),
-                'Link' => ($unlinked) ? false : $this->LinkPages()
-            ]));
-
-            $this->extend('updateBreadcrumbs', $items);
+        if (($this->getAction() !== 'index') && ($record = $this->currentPage())) {
```

Comment:
Checking the action here is necessary because we end up with a record in some of the listview context

Reply:
I'm not sure what you mean, can you please elaborate?

There used to be a condition like this:
```php
$record = $this->currentPage();
if (!$record) {
```

Is this new condition performing that same job? If so why are we doing it differently in this PR than we were before?

Reply:
Yes, this scenario is meant to cover what was covered by
```php
$record = $this->currentPage(); 
if (!$record) {
```

## Scenarios covered by this if statement
`$this->currentPage()` will return a truthy value in the following condition:
- when accessing the page for editing (/admin/pages/edit/show/1)
- when accessing a page's setting (/admin/pages/settings/show/1)
- when accessing a page's history (/admin/pages/history/show/1)

In all tho situation we want the breadcrumb to look like this: 
![image](https://github.com/user-attachments/assets/a1935432-4e98-42d1-a4a6-a7e35b3c87bc)

- The breadcrumb on the right doesn't have a "Pages" root crumb, because that's covered by the "Pages" link at the top of the site tree.
- The crumbs link to the edit form for their page.

## Scenario not covered by this IF
When viewing the listview and you drill down to see the children of the page, that triggers a PJAX request to `/admin/pages?ParentID=123` where `123` is the ID of the page for which you want to view the children.

For that request, `$this->currentPage()` will return an instance of Page ID 123. However, the breadcrumb needs to be different for that use case. Hence why we are adding a `$this->getAction() !== 'index'` check.

That was effectively the root of the original bug:
- First the logic in CMSPagesController::Breadcrumb would be executed, creating a initial breadcrumb.
- Then the logic in this `if` statement would be executed again. Doubling up the breadcrumb.